lib/improver/tests/ensemble_calibration/ensemble_calibration/test_ApplyCoefficientsForEnsembleCalibration.py

```diff
@@ -271,245 +283,169 @@ def test_unmatching_y_axis(self):
-        the calibrated mean is similar to when not using statsmodels and to
-        when the ensemble mean is used as the predictor."""
 
-        optimised_coeffs = dict(
-            zip(self.coeffs_from_statsmodels_realizations.coord(
-                    "coefficient_name").points,
-                self.coeffs_from_statsmodels_realizations.data))
-        predicted_mean = (
-            self.expected_calibrated_predictor_statsmodels_realizations.
-            flatten())
+class Test__create_output_cubes(
+        SetupCoefficientsCubes, EnsembleCalibrationAssertions):
 
-        calibrated_forecast_predictor, calibrated_forecast_var = (
-            self.plugin.calibrate_forecast_data(
-                optimised_coeffs, predicted_mean, self.forecast_predictor,
-                self.forecast_vars))
+    """Test the _create_output_cubes method."""
 
-        self.assertCalibratedVariablesAlmostEqual(
-            calibrated_forecast_predictor.data,
-            self.expected_calibrated_predictor_statsmodels_realizations)
-        self.assertCalibratedVariablesAlmostEqual(
-            calibrated_forecast_var.data,
-            self.expected_calibrated_variance_statsmodels_realizations)
-        assert_array_almost_equal(
-            calibrated_forecast_predictor.data,
-            self.expected_calibrated_predictor_mean,
-            decimal=0)
-        assert_array_almost_equal(
-            calibrated_forecast_predictor.data,
-            self.expected_calibrated_predictor_no_statsmodels_realizations,
-            decimal=0)
-        self.assertIsInstance(calibrated_forecast_predictor, iris.cube.Cube)
-        self.assertIsInstance(calibrated_forecast_var, iris.cube.Cube)
-
-    def test_realization_as_predictor_without_statsmodels(self):
-        """Test that the calibrated forecast using realization as the
-        predictor returns the correct values when not using statsmodels. Check
-        that the calibrated mean is similar to when using statsmodels and to
-        when the ensemble mean is used as the predictor."""
+    def setUp(self):
+        """Set-up the plugin for testing."""
+        super().setUp()
+        self.plugin = Plugin()
+        self.plugin.current_forecast = self.current_temperature_forecast_cube
 
-        optimised_coeffs = dict(
-            zip(self.coeffs_from_no_statsmodels_realizations.coord(
-                    "coefficient_name").points,
-                self.coeffs_from_no_statsmodels_realizations.data))
-        predicted_mean = (
-            self.expected_calibrated_predictor_no_statsmodels_realizations.
-            flatten())
+    @ManageWarnings(
+        ignored_messages=["Collapsing a non-contiguous coordinate."])
+    def test_basic(self):
+        """Test that the cubes created containing the location and scale
+        parameter are formatted as expected."""
+        location_parameter_cube, scale_parameter_cube = (
+            self.plugin._create_output_cubes(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Access to a protected member _create_output_cubes of a client class](https://app.codacy.com/app/metoppv_tech/improver/pullRequest?prid=4714944)